### PR TITLE
add: profile - JoseVale99

### LINKED ISSUE #1672: josevale99
### Profile screenshot

github com_JoseVale99

### PR DESCRIPTION
**Pull request by JoseVale99**
resolves #1672

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [ ] Changes made in this PR are mentioned below
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] I acknowledge that all my contributions will be made under the project's license.
- [ ] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/josevale99.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/39961893/213903888-4347babe-71b8-41b3-b69e-836c1fe87503.png",
```
Comment (Panquesito7) [inline profiles/josevale99.json:2]: I recommend more of an image like this. What do you think? 🙂

image

```suggestion
    "image": "https://user-images.githubusercontent.com/51391473/213935618-d8bbd295-4fee-45cb-94d1-5278624b9042.png",
```

File: profiles/josevale99.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+    "issueId": 1672,
+    "name": "José Valentín",
+    "username": "JoseVale99"
+  }
```
Comment (Panquesito7) [inline profiles/josevale99.json:6]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by JoseVale99 — COMMENTED** [review]
fix json

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**